src/signals/incident/components/form/MapSelectors/Streetlight/StreetlightLayer/StreetlightLayer.tsx

```diff
@@ -18,8 +18,11 @@ export const StreetlightLayer = () => {
     return meta.featureTypes.find(({ typeValue }) => typeValue === 'reported')
   }
 
+  // Only streetlight features that are reported, exclude clocks (objecttype is 1)
   const reportedFeatures = data.features.filter(
-    (feature) => feature.properties?.meldingstatus === 1
+    (feature) =>
+      feature.properties?.objecttype !== 1 &&
```

Comment:
Can we some convert the type ID's to constants instead of having to reference numbers?

Reply:
Or reference `typeField` and `typeValue`?

Reply:
typeField and TypeValue belong to feature type, I've introduced constants to improve the issue